### DISCUSSION TITLE
Use icons without blueprintjs/core?

### DISCUSSION THREAD

**Discussion by switz** [palantir/blueprint#6298] [Q&A]
Is there a good way to _just_ use the `icons` package without @blueprintjs/core?

I think my plan is to do:

```
import DownloadPath from '@blueprintjs/icons/lib/esm/generated/16px/paths/download';

        <svg fill="white" height={16} width={16}>
          {DownloadPath.map((p, i) => (
            <path d={p} key={i} />
          ))}
        </svg>
```

and render these as react server components – so hopefully all I'm introducing into my bundle is the resulting SVG.

**Comment by adidahiya** [comment]
@blueprintjs/icons does not depend on @blueprintjs/core, so I think you should be ok to use the public API of the icons package without using anything from core. Did you run into issues using things like `{ Download } from "@blueprintjs/icons"` without `blueprint.css`? If so, that feels like an implicit dependency which you can file as a bug... I will try to resolve it so you can use the icons independently.
